test/core/require-hook.js

```diff
@@ -0,0 +1,12 @@
+var chai        = require("chai");
```

Comment:
Might be better to put this test [here](https://github.com/babel/babel/tree/master/test/core/fixtures/bin/babel-node) and use a more robust test than just a bare arrow function which will pass if it's successfully ignored and the target environment supports arrow functions.